src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/version-switcher.html

```diff
@@ -7,7 +7,7 @@
   <div class="version-switcher__container dropdown">
     <button id="{{ button_id }}"
       type="button"
-      class="version-switcher__button btn btn-sm navbar-btn dropdown-toggle"
```

Comment:
In most places I replace navbar-btn with pst-navbar-icon, but the version switcher is not an icon button and when I search the codebase for navbar-btn, I get nothing, so it seems safe to remove.

Reply:
was probably cruft